website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -32,8 +32,7 @@ This guide walks through an example of building a simple memcached-operator powe
 Make sure to define `IMG` when you call `make`:
 
   ```sh
-  export OPERATOR_IMG="quay.io/example/memcached-operator:v0.0.1"
-  make docker-build docker-push IMG=$OPERATOR_IMG
+  make docker-build docker-push IMG="example.com/memcached-operator:v0.0.1"
   ```
 
```

Comment:
```suggestion
NOTE: To allow the cluster to pull the image the repository needs to be set as public or you must configure an image pull secret.
```

Reply:
This shouldn't be in the quickstart.

Reply:
We need to add that in the quick start. Otherwise, the users will not understand that the image needs are pushed and pullable. it is a common scenario/issue faced by them following the previous instructions of the operator-sdk-sample that was solved by providing this info.

Reply:
If they are using a private image registry, they definitely know about pull secrets.

Reply:
For me does not matter the place in the doc. However, I think we need to clarify that is required to be able to push and pull the image.